src/client/eppo-client.spec.ts

```diff
@@ -1,9 +1,9 @@
 /**
  * @jest-environment jsdom
  */
-import axios from "axios";
-import * as td from "testdouble";
-import mock, { MockResponse } from "xhr-mock";
```

Comment:
this entire file had formatting issues! I made comments below in the two places there are changes.